src/lib.rs

```diff
@@ -538,9 +543,20 @@ fn apply_patches_musl(target: &str, inner: &Path) {
 
 fn sanitize_sh(path: &Path) -> String {
     if !cfg!(windows) {
-        return path.to_str().unwrap().to_string();
+        return match path.to_str() {
+            Some(value) => value.to_string(),
+            None => {
+                panic!("Failed to convert path to string: {:?}", &path);
+            }
+        };
     }
-    let path = path.to_str().unwrap().replace("\\", "/");
+
+    let path = match path.to_str() {
+        Some(value) => value.replace("\\", "/"),
+        None => {
+            panic!("Failed to convert path to string: {:?}", &path);
+        }
+    };
```

Comment:
Could this and the above `to_str` use a helper function instead of duplicating?

Reply:
Probably